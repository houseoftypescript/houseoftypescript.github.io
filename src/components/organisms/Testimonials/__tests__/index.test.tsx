import { render } from '@testing-library/react';
import { Testimonials } from '../index';

describe('Testimonials', () => {
  it('renders correctly', () => {
    const button = render(<Testimonials testimonials={[]} />);
    expect(button.container).toMatchSnapshot();
  });
});
