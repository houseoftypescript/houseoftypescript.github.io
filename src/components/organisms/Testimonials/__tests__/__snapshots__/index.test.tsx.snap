// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonials renders correctly 1`] = `
<div>
  <section
    class="bg-[#FFF5ED]/50 py-8 md:py-16 testimonials-container"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="rc-tabs rc-tabs-bottom css-0"
      >
        <div
          class="rc-tabs-nav"
          role="tablist"
        >
          <div
            class="rc-tabs-nav-wrap"
          >
            <div
              class="rc-tabs-nav-list"
              style="transform: translate(0px, 0px);"
            >
              <div
                class="rc-tabs-ink-bar rc-tabs-ink-bar-animated"
              />
            </div>
          </div>
          <div
            class="rc-tabs-nav-operations rc-tabs-nav-operations-hidden"
          >
            <button
              aria-controls="rc-tabs-test-more-popup"
              aria-expanded="false"
              aria-haspopup="listbox"
              aria-hidden="true"
              class="rc-tabs-nav-more"
              id="rc-tabs-test-more"
              style="visibility: hidden; order: 1;"
              tabindex="-1"
              type="button"
            >
              More
            </button>
          </div>
        </div>
        <div
          class="rc-tabs-content-holder"
        >
          <div
            class="rc-tabs-content rc-tabs-content-bottom"
          />
        </div>
      </div>
    </div>
  </section>
</div>
`;
